snippets/nordic-rpc-host/boards/nrf54h20dk_nrf54h20_cpurad.overlay

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ #include "nrf54h20dk_nrf54h20-mem-map-move.dtsi"
+
+&mram1x {
+	cpurad_rw_partitions: cpurad-rw-partitions {
```

Comment:
I would move this entry to `nrf54h20dk_nrf54h20-mem-map-move.dtsi` as well. Leaving here only the `status = "okay";` property.

Reply:
Moved, I left `status` and `zephyr,settings_partition`.